website/content/docs/plugins/containerized-plugins.mdx

```diff
@@ -0,0 +1,253 @@
+  runsc requires an additional runtime argument to enable Unix domain socket
+  communication between Vault and the plugin: `--host-uds=all`.
+  For example, if using Docker, `daemon.json` will look like so:
+
+  ```json
+  {
+    "runtimes": {
+      "runsc": {
+        "path": "/usr/local/bin/runsc",
+        "runtimeArgs": [
+          "--host-uds=all"
+        ]
+      }
+    }
+  }
+  ```
+
+* Any plugin container images must be pulled and already available locally.
+  Vault does not currently support pulling images as part of the plugin
+  registration process.
+
+## Running containerized plugins
+
+### Installation
+
+1. Install a container engine, for example:
+    * [Docker](https://docs.docker.com/engine/install/)
+    * Or [rootless Docker](https://docs.docker.com/engine/security/rootless/)
+
+2. [Install runsc](https://gvisor.dev/docs/user_guide/install/)
+
+3. Configure runsc to allow Unix sockets between the host and plugin, updating
+   the runsc install location if you didn't use the default:
+
+    a. Default Docker install:
+        ```bash
+        sudo tee /etc/docker/daemon.json <<EOF
+        {
+        "runtimes": {
+          "runsc": {
+            "path": "/usr/local/bin/runsc",
+              "runtimeArgs": [
+                "--host-uds=all"
+              ]
+            }
+          }
+        }
+        EOF
+        sudo systemctl reload docker
+        ```
+    
+    b. Rootless Docker install:
+        ```bash
+        mkdir -p ~/.config/docker
+        tee ~/.config/docker/daemon.json  <<EOF
+        {
+          "runtimes": {
+            "runsc": {
+              "path": "/usr/local/bin/runsc",
+              "runtimeArgs": [
+                "--host-uds=all"
+                "--ignore-cgroups"
+              ]
+            }
+          }
+        }
+        EOF
+        systemctl --user restart docker
+        ```
+
+### Building plugin container images
+
+To be compatible with running in a container, plugins must be built with at
+least v1.5.0 of [go-plugin](https://github.com/hashicorp/go-plugin). This can
+be pulled in directly via `go get github.com/hashicorp/go-plugin@latest`, or
+through the SDK from at least v0.10.0: `go get github.com/hashicorp/vault/sdk@latest`.
+
+Plugin container images should run a binary the same way it would be expected
+to run as a plain plugin process. For most plugins this simply means running
+the binary as the entrypoint. For example, for the KV plugin:
+
+1. Build the binary
+    ```bash
+    git clone https://github.com/hashicorp/vault-plugin-secrets-kv.git
+    cd vault-plugin-secrets-kv/
+    GOOS=linux go build -o kv cmd/vault-plugin-secrets-kv/main.go
+    ```
+
+1. Create the following `Dockerfile`:
+    ```Dockerfile
+    FROM gcr.io/distroless/static-debian12
+
+    COPY kv /bin/kv
+
+    ENTRYPOINT [ "/bin/kv" ]
+    ```
+
+1. Build the image:
+    ```bash
+    docker build -t hashicorp/vault-plugin-secrets-kv .
+    ```
```

Comment:
````suggestion
1. Build the image:
    ```shell-session
    docker build -t PATH/TO/THE/PLUGIN/BINARY .
    ```
````

I'm not entirely clear from the previous steps what the build path is supposed to be. It it a path to the compiled binary or something else? For what it's worth, this is why generalizing is useful for example code, it helps folks note how the commands are related :)

Style correction: use `shell-session` for bash code, generalize example commands when possible

Reply:
@schavis `hashicorp/vault-plugin-secrets-kv` is the "tag" being given to the image by the `docker build -t` command (-t for tag). The tag is a human-readable ID given to the image. The build path is specified by the `.` which means the current working directory.